### PR TITLE
Docsteam 112/server

### PR DESCRIPTION
**Pull request by catherinepope**
# Description
- Fixed typos.
- Applied style guide.
- Tidied formatting.

# Reasons
https://circleci.atlassian.net/browse/DOCSTEAM-112

### CODE REVIEW

File: jekyll/_cci2/server-3-install-build-services.adoc [hunk 1]
```diff
@@ -678,31 +678,31 @@ gcloud projects add-iam-policy-binding <YOUR_PROJECT_ID> --member serviceAccount
...
+Configure VM Service through the KOTS Admin Console. Details of the available configuration options can be found in the https://circleci.com/docs/2.0/server-3-operator-vm-service[VM Service] guide.
 +
 Once you have configured the fields, *save your config* and deploy your updated application.
 
 endif::env-aws[]
 
 ==== VM Service Validation
 
-Once you have configured and deployed CircleCI server you should validate that VM Service is operational. You can rerun the reality checker project within your CircleCI installation and you should see the VM Service Jobs complete with green. At this point all tests should pass with green.
+Once you have configured and deployed CircleCI server, you should validate that VM Service is operational. You can rerun the realitychecker project within your CircleCI installation and you should see the VM Service Jobs complete with green. At this point, all tests should pass with green.
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-install-build-services.adoc:697]: "with green" sounds really weird to me. Pass is enough I think. What do you think @catherinepope?

File: jekyll/_cci2/server-3-install-hardening-your-cluster.adoc [hunk 2]
```diff
@@ -92,13 +88,13 @@ If you want to access your instances using SSH, you will need to open port 22 fo
...
+Instances building jobs (that is, the Nomad clients and external VMs) also will likely need to pull code from your VCS using SSH (TCP port 22). SSH is also used to communicate with external VMs, so it should be allowed for all instances with the destination of the VM subnet and your VCS, at the very least.
 
 ## Kubernetes Nodes
 
 ### Intra-node traffic
-The traffic within your K8s cluster is regulated by networking policies by default. For most purposes, this should be sufficient to regulate the traffic between pods and there is no additional requirement to reduce traffic between K8s nodes any further (i.e. it is fine to allow all traffic between K8s nodes).
+By default, the traffic within your Kuberntes cluster is regulated by networking policies. For most purposes, this should be sufficient to regulate the traffic between pods and there is no additional requirement to reduce traffic between Kuberntes nodes any further (it is fine to allow all traffic between Kubernetes nodes).
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-install-hardening-your-cluster.adoc:97]:
```suggestion
By default, the traffic within your Kubernetes cluster is regulated by networking policies. For most purposes, this should be sufficient to regulate the traffic between pods and there is no additional requirement to reduce traffic between Kubernetes nodes any further (it is fine to allow all traffic between Kubernetes nodes).
```

File: jekyll/_cci2/server-3-operator-backup-and-restore.adoc [hunk 3]
```diff
@@ -17,20 +17,21 @@ toc::[]
...
-The benefit of this approach is that it not only restores your application's data,
+CircleCI server is administered via https://kots.io/[KOTS], which uses https://velero.io/[Velero] for backup and restore. The benefit of this approach is that it not only restores your application's data,
 but it also restores the state of the Kubernetes cluster and its resources at the time of the backup.
-In this way, we can also restore admin-console configurations and customizations you made to your cluster. 
+This means you can also restore Admin Console configurations and customizations you made to your cluster. 
 
 NOTE: Backup and restore of the CircleCI services is dependent on Velero. If your cluster is lost, you will not be able to restore CircleCI until you have successfully started Velero in the cluster. From there you can recover the CircleCI services.
 
-== The setup
+== The Setup
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-operator-backup-and-restore.adoc:30]: should be sentence case - but I see all the headings are the wrong case in this doc!

File: jekyll/_cci2/server-3-operator-backup-and-restore.adoc [hunk 4]
```diff
@@ -60,7 +61,7 @@ Below, you will find instructions for creating a server 3.x backup on AWS and GC
 ////
 == Server 3.x backups on AWS
 
-The following steps will assume AWS as your provider and you have met the <<prerequisites, prerequisites>> listed above.
+The following steps assume AWS as your provider and that you have met the <<prerequisites, prerequisites>> listed above.
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-operator-backup-and-restore.adoc:64]: I'm not 100% but I think this crossref can just be <<prerequisites>>

File: jekyll/_cci2/server-3-operator-backup-and-restore.adoc [hunk 5]
```diff
@@ -376,7 +375,7 @@ As restic is a daemonset, there should be one pod for each node in your Kubernet
 ////
 == Server 3.x backups with S3 Compatible Storage
 
-The following steps will assume you arre using S3-compatible object storage, but not necessarily AWS S3, for your backups.
+The following steps assume you arre using S3-compatible object storage, but not necessarily AWS S3, for your backups.
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-operator-backup-and-restore.adoc:378]:
```suggestion
The following steps assume you are using S3-compatible object storage, but not necessarily AWS S3, for your backups.
```

File: jekyll/_cci2/server-3-operator-backup-and-restore.adoc [hunk 6]
```diff
@@ -541,15 +539,15 @@ kubectl kots restore --from-backup <backup-instance-id>
...
+IMPORTANT: The restore will create new load balancers for CircleCI's services. Consequently, you will need to either update your DNS records or the hostname/IP configurations in KOTS Admin Console. You may also need to consider updating the `nomad server endpoint` provided to your Nomad clients.
 
-IMPORTANT: If you are using pre-existing Nomad clients, you will need to restart them before they will connect to the nomad-server cluster.
+IMPORTANT: If you are using pre-existing Nomad clients, you need to restart them before they will connect to the nomad-server cluster.
 
-It should take roughly 10-15 mins for CircleCI server to be restored and operational.
+It should take approximately 10-15 mins for CircleCI server to be restored and operational.
 
-== Optional - Scheduling backups with kots
+== Optional - Scheduling backups with KOTs
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-operator-backup-and-restore.adoc:550]:
```suggestion
== Optional - Scheduling backups with KOTS
```

File: jekyll/_cci2/server-3-operator-backup-and-restore.adoc [hunk 7]
```diff
@@ -572,15 +570,15 @@ $ kubectl kots version
...
 
 ```
 $ kubectl logs deployment/velero --namespace velero
 ```
 
-You may need to reinstall Velero as a result.
+You may need to reinstall Velero afterwards.
 
-* Confirm that snapshots are available on your license. You may reach out to our Customer Support Team for confirmation.
+* Confirm that snapshots are available on your license. You may contact our Customer Support Team for confirmation.
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-operator-backup-and-restore.adoc:581]: :-D

File: jekyll/_cci2/server-3-operator-build-artifacts.adoc [hunk 8]
```diff
@@ -6,7 +6,7 @@ version:
 = CircleCI Server v3.x build artifacts
 :page-layout: classic-docs
 :page-liquid:
-:page-description: Learn how build artifacts persist data after a job is completed and how they can be used for longer-term storage of your build process outputs.
+:page-description: Learn how to build artifacts persist data after a job is completed and how they can be used for longer-term storage of your build process outputs.
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-operator-build-artifacts.adoc:9]:
```suggestion
:page-description: Learn how build artifacts persist data after a job is completed and how they can be used for longer-term storage of your build process outputs.
```

File: jekyll/_cci2/server-3-operator-vm-service.adoc [hunk 9]
```diff
@@ -20,7 +20,7 @@ toc::[]
 NOTE: We recommend that you leave these options at their defaults until you have successfully configured and verified the core and build services of your server installation. Steps to set up VM service are provided in the server 3.x installation guide for https://circleci.com/docs/2.0/server-3-install-build-services/#eks[AWS] and https://circleci.com/docs/2.0/server-3-install-build-services/#gke[GCP].
 
 == VM service settings
-You will need to provide the hostname/IP for your VM service load balancer:
+You  need to provide the hostname/IP for your VM service load balancer:
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-operator-vm-service.adoc:23]:
```suggestion
You need to provide the hostname/IP for your VM service load balancer:
```

**Review by rosieyohannan — COMMENTED** [review]
Thank you so much for this extensive review!!!

File: jekyll/_cci2/server-3-install-build-services.adoc (hunk 1 above)
Comment (catherinepope) [reply jekyll/_cci2/server-3-install-build-services.adoc]: I agree!

File: jekyll/_cci2/server-3-operator-backup-and-restore.adoc (hunk 3 above)
Comment (catherinepope) [reply jekyll/_cci2/server-3-operator-backup-and-restore.adoc]: Haha, I've made them all sentence case.

**Pull request merged**